definitions/npm/react-redux_v5.x.x/flow_v0.89.x-/test_connect.js

```diff
@@ -722,3 +724,23 @@ function checkIfStateTypeIsRespectedAgain() {
   <Connected />;
   e.push(Connected);
 }
+
+function testPassingDispatchPropWithoutDispatchFunction() {
+  type OwnProps = {||}
+  type Props = {| ...OwnProps, dispatch: any |};
```

Comment:
I'd also add a test with an exact type for `dispatch` to test that the typings pass the correct type explicitly. I haven't seen any tests for this case in the pre-0.89 version and neither added them myself.

Reply:
√  Also tested that `dispatch` is not passed when `mapDispatchToProps` function is provided.